chain/neutrino.go

```diff
@@ -92,12 +93,17 @@ func (s *NeutrinoClient) Start() error {
 // Stop replicates the RPC client's Stop method.
 func (s *NeutrinoClient) Stop() {
 	s.clientMtx.Lock()
-	defer s.clientMtx.Unlock()
 	if !s.started {
+		s.clientMtx.Unlock()
```

Comment:
The defer can likely be left as is, don't see anything performance critical here.

Reply:
Done